.travis.yml

```diff
@@ -17,29 +17,40 @@ addons:
 sudo: required
 
 php:
-  - 8.2.0
   - 8.1.0
+  - 8.2.0
+
 env:
-  - APPLICATION=ojs BRANCH=main TEST=mysql
-  - APPLICATION=ojs BRANCH=main TEST=pgsql
+  - APPLICATION=ojs BRANCH=main TEST=mysql DBTYPE=MySQL
```

Comment:
Ah, I merged this in stable-3_4_0, but wanted to ask where this DBTYPE is needed?

Reply:
Instead of installing OJS, this updated build script is using a ready database from the [datasets](https://github.com/pkp/datasets) repository, and this variable is used by the scripts at https://github.com/pkp/datasets/blob/main/tools/.